tests/fulldecay/test_mass_functions.py

```diff
@@ -0,0 +1,56 @@
+from typing import Callable
+
+import pytest
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+import phasespace.fulldecay.mass_functions as mf
+
+KSTARZ_MASS = 895.81

```

Comment:
From the latest, 2021, PDG I actually get slightly different numbers:

```
In [3]: print(Particle.from_pdgid(313).describe())
Name: K*(892)0       ID: 313          Latex: $K^{*}(892)^{0}$
Mass  = 895.55 ± 0.20 MeV
Width = 47.3 ± 0.5 MeV
Q (charge)        = 0       J (total angular) = 1.0      P (space parity) = -
C (charge parity) = None    I (isospin)       = 0.5      G (G-parity)     = None
    SpinType: SpinType.Vector
    Quarks: dS
    Antiparticle name: K*(892)~0 (antiparticle status: Barred)
```

Why not use those, comment on the fact that they are PDG 2021 numbers, and also add as a comment the units (always a good thing)?

Reply:
I think I just copied these values from a [phasespace example](https://phasespace.readthedocs.io/en/stable/usage.html#resonances-with-variable-mass) but I agree that it is probably better to just use the mass values from `particle`.

Reply:
I updated the README very recently but it may well be that more updates are "needed" elsewhere. To be looked at :-).

Reply:
Fixed in 60ab98243c8cf265604594db78b28d46bfd824fd